packages/example/src/hooks/useLockModal.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect } from 'react';
```

Comment:
you should put the hook with its modal, don't mind the `src/components` vs `src/hooks` separation it's not ideal I think